feature-libs/asm/components/customer-list/customer-list.component.html

```diff
@@ -0,0 +1,354 @@
+                              "
+                            ></ng-template>
+                          </ng-container>
+                        </ng-container>
+                      </td>
+                    </ng-container>
+                    <!-- one column if mobile -->
+                    <td
+                      role="cell"
+                      class="cx-multi-cell"
+                      *ngIf="(breakpoint$ | async) === BREAKPOINT.xs"
+                    >
+                      <ng-container
+                        *ngFor="let column of customerListConfig?.columns"
+                      >
+                        <ng-template
+                          *ngTemplateOutlet="
+                            cell;
+                            context: {
+                              customerEntry: customerEntry,
+                              column: column,
+                              showHeader: true
+                            }
+                          "
+                        ></ng-template>
+                      </ng-container>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+              <div
+                class="cx-empty-state"
+                *ngIf="!customerSearchPage?.entries.length"
+              >
+                {{ 'asm.customerList.noCustomers' | cxTranslate }}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </ng-container>
+  </div>
+</div>
+
+<ng-template
+  #cell
+  let-customerEntry="customerEntry"
+  let-column="column"
+  let-showHeader="showHeader"
+>
+  <div class="cx-cell-container">
+    <span class="cx-header-text" *ngIf="showHeader">
+      {{ column.headerLocalizationKey | cxTranslate }}
+    </span>
+
+    <ng-container *ngIf="!column.actionType">
+      <ng-container
+        *ngTemplateOutlet="
+          cellContent;
+          context: { column: column, customerEntry: customerEntry }
+        "
+      ></ng-container>
+    </ng-container>
+
+    <button
+      *ngIf="column.actionType"
+      (click)="startColumnAction(customerEntry, column.actionType)"
+      class="btn btn-link cx-action-link cx-btn-cell"
+      [attr.title]="
+        column.icon
+          ? (column.icon.captionLocalizationKey ?? column.headerLocalizationKey
+            | cxTranslate)
+          : undefined
+      "
+      [attr.aria-label]="
+        column.icon
+          ? (column.icon.captionLocalizationKey ?? column.headerLocalizationKey
+            | cxTranslate)
+          : undefined
+      "
+    >
+      <ng-container
+        *ngTemplateOutlet="
+          cellContent;
+          context: { column: column, customerEntry: customerEntry }
+        "
+      ></ng-container>
+    </button>
+  </div>
+</ng-template>
+<ng-template #cellContent let-customerEntry="customerEntry" let-column="column">
+  <span *ngIf="!column.icon">{{
+    column.renderer ? column.renderer(customerEntry) : ''
```

Comment:
I want to say Angular supports the Elvis operator in template HTML now
```suggestion
    column.renderer?.(customerEntry) || ''
```